### DISCUSSION TITLE
SYMMETRY TOOL

### DISCUSSION THREAD

**Discussion by LoneStarz** [keiwando/toonsquid#38] [Ideas]
Sorry for making so many of these but yes I just realized a symmetry tool would be really useful.

**Comment by keiwando** [comment]
No worries, that's pretty much the point of this forum :) I want everyone to post all of their feature requests here for others to also see and upvote or discuss, which helps me to get a better idea of what to prioritize in the future. I always see and keep track of every post here, even if I might not have the time to reply to all.
